src/blockchain_processor.py

```diff
@@ -348,6 +352,11 @@ def add_to_history(self, addr, tx_hash, tx_pos, tx_height):
         self.batch_txio[txo] = addr
 
 
+    def get_masternodes_status(self):
+        return self.dashd('masternode', ['list'])
+
+    def get_proposals_status(self):
+        return self.dashd('mnbudget', ['list'])
```

Comment:
...which is weird but true for 12.0
And it is going to change to `mngovernance list` in 12.1 😉
https://github.com/dashpay/dash/blob/v0.12.1.x/src/rpcmasternode-budget.cpp#L110